packages/vapor/scss/components/filepicker.scss

```diff
@@ -0,0 +1,8 @@
+input.filepicker {
+    position: absolute;
+    z-index: -1;
+    width: 0.1px;
+    height: 0.1px;
```

Comment:
weird width and height 🤔

Reply:
> Setting the property values to zero ends up throwing the element out of tab party in some browsers. And position: absolute guarantees the element does not interfere with the sibling elements.
https://tympanus.net/codrops/2015/09/15/styling-customizing-file-inputs-smart-way/